static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -301,8 +308,8 @@ export default function GroupSidebar({
       {issueTypeConfig.regression.enabled && event && (
         <EventThroughput event={event} group={group} />
       )}
-      {renderParticipantData()}
-      {renderSeenByList()}
+      {!hasStreamlinedUI && renderParticipantData()}
+      {!hasStreamlinedUI && renderSeenByList()}
```

Comment:
i already followed up to figure out how we want to view all for this, but decided to leave that for a future PR to limit scope. i think in this scenario because the designs are so different it might make more sense to rebuild them in header (since it really is just a `StyledAvatarList`)